.circleci/config.yml

```diff
@@ -0,0 +1,96 @@
+default: &defaults
+  docker:
+    - image: circleci/ruby:2.6.1-node-browsers
+      environment:
+        PGHOST: 127.0.0.1
+        PGUSER: postgres
+        RAILS_ENV: test
+        CC_TEST_REPORTER_ID: ${CC_TEST_REPORTER_ID}
+    - image: postgres:9.5
+      environment:
+        POSTGRES_USER: postgres
+        POSTGRES_DB: post-api_test
+        POSTGRES_PASSWORD: ""
+
+  working_directory: ~/post-api-revamped
+
+# Restore Cached Dependencies
+cmd_restore_cache: &cmd_restore_cache
+  restore_cache:
+      key: post-api-revamped-{{ checksum "Gemfile.lock" }}
+
+# Bundle install dependencies
+cmd_install_dependencies: &cmd_install_dependencies
+  run:
+    name: Install dependencies
+    command: |
+      echo 'export BUNDLER_VERSION=$(cat Gemfile.lock | tail -1 | tr -d " ")' >> $BASH_ENV
+      source $BASH_ENV
+      gem install bundler
+      bundle install --path vendor/bundle
+
+# Cache Dependencies
+cmd_save_cache: &cmd_save_cache
+  save_cache:
+    key: post-api-revamped-{{ checksum "Gemfile.lock" }}
+    paths:
+      - vendor/bundle
+
+# Wait for DB
+cmd_wait_for_postgres: &cmd_wait_for_postgres
+  run:
+    name: Waiting for Postgres to be ready
+    command: |
+      dockerize -wait tcp://localhost:5432 -timeout 1m
+
+cmd_attach_workspace: &cmd_attach_workspace
+  attach_workspace:
+    at: ~/post-api-revamped
+
+version: 2
+jobs:
+  build:
+    <<: *defaults
+    steps:
+      - checkout
+      - *cmd_restore_cache
+      - *cmd_install_dependencies
+      - *cmd_save_cache
+      - *cmd_wait_for_postgres
+      - run:
+          name: Database setup
+          command: |
+             bundle exec rake db:drop
+             bundle exec rake db:create
+             bundle exec rake db:migrate
+             bundle exec rake db:schema:load --trace
+
+  test:
+    <<: *defaults
+    steps:
+      - checkout
+      - *cmd_restore_cache
+      - *cmd_install_dependencies
+      - *cmd_save_cache
+      - *cmd_wait_for_postgres
+      - run:
+          name: Setup Code Climate test-reporter
+          command: |
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            chmod +x ./cc-test-reporter
+      - run:
+          name: Run tests
+          command: |
+            ./cc-test-reporter before-build
+            bundle install
+            bundle exec rspec
+            ./cc-test-reporter after-build --exit-code $?
```

Comment:
Hey, I am guessing this is what is making our build fail on CircleCI, or what's your thought?

Reply:
L87 reports the test coverage to Codeclimate